packages/ui-lib/src/Alerts/atom/Tooltip.tsx

```diff
@@ -0,0 +1,269 @@
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+`;
+
+const getDirectionStyle = (state: ITooltipPosition, coords: DOMRect) => {
+
+  const { left, top, width, height } = coords;
+
+  const currentTop = top + window.scrollY;
+  const currentLeft = left + window.scrollX;
+
+  switch (state) {
+    case 'bottom-end':
+      return `
+              flex-direction: column;
+              top: ${currentTop + height}px;
+              left: ${currentLeft - ARROW_SIZE - ARROW_MARGIN + (width / 2)}px;
+              ${ARROW} {
+                margin-left: ${ARROW_MARGIN}px;
+              }
+            `;
+
+    case 'bottom-center':
+      return `
+              flex-direction: column;
+              top: ${currentTop + height}px;
+              left: ${currentLeft + (width / 2)}px;
+              transform: translateX(-50%);
+              align-items: center;
+            `;
+
+    case 'bottom-start':
+      return `
+              flex-direction: column;
+              align-items: end;
+              top: ${currentTop + height}px;
+              left: ${currentLeft + ARROW_SIZE + ARROW_MARGIN + (width / 2)}px;
+              transform: translateX(-100%);
+              ${ARROW} {
+                margin-right: 16px;
+              }
+            `;
+
+    case 'right':
+      return `
+              flex-direction: row;
+              align-items: center;
+              top: ${currentTop + (height / 2)}px;
+              left: ${currentLeft + width}px;
+              transform: translateY(-50%);
+              ${ARROW} {
+                transform: rotate(-90deg);
+                margin-right: -4px;
+              }
+            `;
+
+    case 'left':
+      return `
+              flex-direction: row-reverse;
+              align-items: center;
+              top: ${currentTop + (height / 2)}px;
+              left: ${currentLeft}px;
+              transform: translate(-100%, -50%);
+              ${ARROW} {
+                transform: rotate(90deg);
+                margin-left: -4px;
+              }
+            `;
+
+    case 'top-end':
+      return `
+              flex-direction: column-reverse;
+              top: ${currentTop}px;
+              left: ${currentLeft - ARROW_SIZE - ARROW_MARGIN + (width / 2)}px;
+              transform: translateY(-100%);
+              ${ARROW} {
+                transform: rotate(180deg);
+                margin-left: ${ARROW_MARGIN}px;
+              }
+            `;
+
+    case 'top-start':
+      return `
+              flex-direction: column-reverse;
+              align-items: end;
+              top: ${currentTop}px;
+              left: ${currentLeft + ARROW_SIZE + ARROW_MARGIN + (width / 2)}px;
+              transform: translate(-100%, -100%);
+              ${ARROW} {
+                transform: rotate(180deg);
+                margin-right: 16px;
+              }
+            `;
+
+    case 'top-center':
+
+    // eslint-disable-next-line no-fallthrough
```

Comment:
Yes, the disable is because I'm allowing that 'top-center' to be the 'default' without duplicating code. I though it be good to have it explicit but I can remove the case and leave it as a comment if you prefer.